googlefirestore/src/main/java/com/yahoo/ycsb/db/package-info.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2019 YCSB contributors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you
+ * may not use this file except in compliance with the License. You
+ * may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License. See accompanying
+ * LICENSE file.
+ */
+
+/**
+ * The YCSB binding for Google's Cloud Firestore.
```

Comment:
classes should be in a binding-specific package. e.g. `com.yahoo.ycsb.db.firestore`

Reply:
this has been moved.